src/components/node-list/node-list-row.test.js

```diff
@@ -144,6 +144,28 @@ describe('NodeListRow', () => {
           wrapper.find('.pipeline-row__toggle-icon--focus-checked').exists()
         ).toBe(true);
       });
+      it('hides the visibility toggle when the row is selected for focus mode', () => {
```

Comment:
Nit: try and put a blank line before `it()` statements like is done elsewhere in this file.